src/library/autodj/dlgautodj.cpp

```diff
@@ -256,6 +256,11 @@ void DlgAutoDJ::skipNextButton(bool) {
     m_pAutoDJProcessor->skipNext();
 }
 
+void DlgAutoDJ::addRandomTrackButton(bool) {
```

Comment:
This can be removed. Functions declared in the singnal block are created be the Qt moc tool.